### PR TITLE
Feature/implement guassian bkim

### PR DESCRIPTION
**Pull request by yrkim98**
Fished MPP:
https://github.com/AllenCell/napari-aicssegmentation/issues/8

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -16,6 +16,7 @@ def read(fname):
 requirements = [
     "napari-plugin-engine>=0.1.4",
     "numpy",
+    "aicssegmentation",
```
Comment (jxchen01) [inline setup.py:19]: I would suggest to add version>=0.1.22

**Review by jxchen01 — DISMISSED** [review]

**Comment by heeler** [comment]
This is nice work. It is clean to read and such. The one thing I would recommend is linting. Flake8/black are good choices. I'll add an automated lint check to the project. It just keeps things readable as code get's more involved.

**Comment by toloudis** [comment]
> This is nice work. It is clean to read and such. The one thing I would recommend is linting. Flake8/black are good choices. I'll add an automated lint check to the project. It just keeps things readable as code get's more involved.

I thought cookiecutter brought in flake8 and black.  If it doesn't, it should.

**Comment by heeler** [comment]
> > This is nice work. It is clean to read and such. The one thing I would recommend is linting. Flake8/black are good choices. I'll add an automated lint check to the project. It just keeps things readable as code get's more involved.
> 
> I thought cookiecutter brought in flake8 and black. If it doesn't, it should.

Ours does but the napari cookie-cutter doesn't. It's easy enough to add, it just slipped through in my eagerness to setup the repo.

File: napari_aicssegmentation/_dock_widget.py [hunk 2]
```diff
@@ -1,36 +1,53 @@
...
         self.viewer = napari_viewer
 
-        btn = QPushButton("Click me!")
-        btn.clicked.connect(self._on_click)
+        btn = QPushButton("Gaussian kernel size = 3.0")
+        btn.clicked.connect(self.smooth_image)
+
+        desc = QLabel("Click button to smooth the current viewport image, higher numbers blur more. Result is displayed as a new channel.")
+        desc.setWordWrap(True)
+        # desc.setAlignment(Qt.AlignCenter)
```
Comment (schoinh) [inline napari_aicssegmentation/_dock_widget.py:21]: We can get rid of this comment, I forgot to delete it

**Review by Aerendel — APPROVED** [review]

**Review by schoinh — APPROVED** [review]

**Pull request merged**